src/Service/Encrypt.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Ambta\DoctrineEncryptBundle\Service;
+
+use Ambta\DoctrineEncryptBundle\Encryptors\EncryptorInterface;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\MySQL80Platform;
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * Doctrine event service which encrypt/decrypt entities.
+ */
+class Encrypt
```

Comment:
As a personal rule, I prefer if the type of class is added as a suffix to make it easier to recognise in other classes.

I propose renaming this to `EncryptService`.